sequencing-server/src/lib/codegen/CommandEDSLPreface.ts

```diff
@@ -545,7 +597,13 @@ export class Sequence implements SeqJson {
       ...(this.steps
         ? {
             steps: this.steps.map(step => {
-              if (step instanceof CommandStem || step instanceof Ground_Block || step instanceof Ground_Event)
+              if (
+                  step instanceof CommandStem ||
+                  step instanceof Ground_Block ||
+                  step instanceof Ground_Event ||
+                  step instanceof ActivateStep ||
+                  step instanceof LoadStep
```

Comment:
When do we decide to use capitalize types vs underscore them?

Reply:
GroundBlock and GroundEvent namespace were already taken by the spec so I used an underscore.